src/averageAges.js

```diff
@@ -14,12 +14,17 @@
  *
  * @return {number}
  */
+
+const forAver = (array) =>
```

Comment:
```suggestion
const getAverageAge = (people) =>
```

Reply:
rename your function, like on the given example, the name should describe what it does